### PR TITLE
Update expired link in quickstart.md

### PR DESCRIPTION
**Pull request by ZeddYu**

### CODE REVIEW

File: doc/doc_ch/quickstart.md [hunk 1]
```diff
@@ -43,7 +43,7 @@
 pip install paddleocr
 ```
 
-- 对于Windows环境用户：直接通过pip安装的shapely库可能出现`[winRrror 126] 找不到指定模块的问题`。建议从[这里](https://www.lfd.uci.edu/~gohlke/pythonlibs/#shapely)下载shapely安装包完成安装。
+- 对于Windows环境用户：直接通过pip安装的shapely库可能出现`[winRrror 126] 找不到指定模块的问题`。建议从[这里](https://pypi.org/project/shapely/#files)下载shapely安装包完成安装。
```
Comment (GreatV) [inline doc/doc_ch/quickstart.md:46]: The package  `shapely` for window is provided by default after python 3.7, so this entry can just be removed.

Comment (GreatV) [reply doc/doc_ch/quickstart.md:46]: Please remove the English description as well.

**Review by GreatV — APPROVED** [review]
LGTM

**Comment by luotao1** [comment]
@ZeddYu Thanks for your contribution! You will receive a beautiful PaddlePaddle gift. Please provide your mailing address by filling out the following questionnaire before October 18th.

Looking forward to the future, we will walk further together in the world of open source!
Click Here ：https://paddle.wjx.cn/vm/h4On9gJ.aspx#

**Comment by luotao1** [comment]
hi, @ZeddYu 
- 非常感谢你对飞桨的贡献，我们正在运营一个PFCC组织，会通过定期分享技术知识与发布开发者主导任务的形式持续为飞桨做贡献，详情可见 https://github.com/luotao1 主页说明。
- 如果你对PFCC有兴趣，请发送邮件至 [ext_paddle_oss@baidu.com](mailto:ext_paddle_oss@baidu.com)，我们会邀请你加入~

**Pull request merged**